bson.cabal

```diff
@@ -36,6 +36,7 @@ Library
                     , data-binary-ieee754
                     , mtl >= 2
                     , text >= 0.11
+		    , fail
```

Comment:
use spaces instead of tabs & travis will probably stop complaining.

Also thx for the patch; i have to try it